rlw.sh

```diff
@@ -98,7 +98,7 @@ rwinetricks () {
 roblox-install () {
 	printf '%b\n' " > begin roblox-install ()\n---"
 	if [[ ! -d "$WINEPREFIX/drive_c" ]]; then
-		spawndialog question 'A working Roblox wineprefix was not found.\nWould you like to install one?'
+		spawndialog question 'Roblox Linux Wrapper doesnt seem to be installed.\nWould you like to install it?'
 		if [[ $? = "0" ]]; then
```

Comment:
Installation of the script in this case is not the same as creating the wineprefix. It's better to be specific in these messages.